src/Stormpath/Util/UserAgentBuilder.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+
+namespace Stormpath\Util;
+
+use Stormpath\Exceptions\UserAgentException;
+
+class UserAgentBuilder {
+
+    /**
+     * @var string
+     */
+    protected $sdkName = 'stormpath-sdk-php';
```

Comment:
If we have a default value on the setter, we don't need to initialize this here.